pandas/tests/io/parser/usecols.py

```diff
@@ -481,7 +481,7 @@ def test_raise_on_usecols_names_mismatch(self):
         data = 'a,b,c,d\n1,2,3,4\n5,6,7,8'
 
         if self.engine == 'c':
-            msg = 'Usecols do not match names'
+            msg = "do not match columns, columns expected but not found"
```

Comment:
I would prefer that you explicitly put in the string which elements are missing.  If that's what you're going for, then go for it!